### ISSUE TITLE
Adguard-Home: Too many Open Files

### ISSUE THREAD

**Issue by alxbl** [alxbl/lab#23]
AdGuard-Home deployment won't start anymore because of inotify.

https://stackoverflow.com/a/63700455
https://kind.sigs.k8s.io/docs/user/known-issues/#pod-errors-due-to-too-many-open-files
https://github.com/kubernetes-sigs/kind/issues/2586
https://superuser.com/questions/1779937/whats-the-maximum-safe-limit-for-fs-inotify-max-user-watches

This works:

```sh
# On the host
sudo sysctl fs.inotify.max_user_watches=524288
sudo sysctl fs.inotify.max_user_instances=512
```

But it needs to be made permanent as part of node provisioning.

**Comment by alxbl** [comment]
Hopefully fixed by 9acd4f4

**Comment by alxbl** [comment]
The problem turned out to be lidarr having too many monitored audio files (which were missing metadata) IIRC.

I've removed lidarr (not using it anyway) and disabled jellyfin's live monitoring. Handles are back under control. Will investigate if I ever need lidarr or if I need jellyfin live monitoring.
